contentcuration/contentcuration/utils/recommendations.py

```diff
@@ -0,0 +1,48 @@
+from automation.utils.appnexus.base import Adapter
+from automation.utils.appnexus.base import Backend
+from automation.utils.appnexus.base import BackendFactory
+
+
+class RecommendationsBackendRequest(object):
+    pass
+
+
+class RecommedationsRequest(RecommendationsBackendRequest):
+    def __init__(self) -> None:
+        super().__init__()
+
+
+class EmbeddingsRequest(RecommendationsBackendRequest):
+    def __init__(self) -> None:
+        super().__init__()
+
+
+class RecommendationsBackendResponse(object):
+    pass
+
+
+class RecommendationsResponse(RecommendationsBackendResponse):
+    def __init__(self) -> None:
+        pass
+
+
+class EmbeddingsResponse(RecommendationsBackendResponse):
+    def __init__(self) -> None:
+        pass
+
+
+class RecommendationsBackendFatory(BackendFactory):
```

Comment:
```suggestion
class RecommendationsBackendFactory(BackendFactory):
```

Reply:
Nice spot @akash5100. I will make the changes to this